php70/Dockerfile

```diff
@@ -3,7 +3,8 @@ FROM centos:7
 # Install base packages.
 RUN yum -y install epel-release centos-release-scl yum-plugin-ovl deltarpm && \
     yum -y update && \
-    yum -y install sudo ssh curl less vim-minimal dnsutils openssl
```

Comment:
We probably shouldn't be supporting/building this any longer either, since PHP 7.0 went out of support before PHP 5.6.